src/nfa/nfa-inclusion.cc

```diff
@@ -215,11 +212,10 @@ bool Mata::Nfa::Algorithms::is_included_antichains(
 namespace {
     using AlgoType = decltype(Algorithms::is_included_naive)*;
 
-    bool compute_equivalence(const Nfa &lhs, const Nfa &rhs, const Mata::Alphabet *const alphabet, const StringMap &params,
-                             const AlgoType &algo) {
+    bool compute_equivalence(const Nfa &lhs, const Nfa &rhs, const Mata::Alphabet *const alphabet, const AlgoType &algo) {
```

Comment:
What is this function? What is the fifference form compute_equivalence?

Reply:
Part of issue #113, not the focus of this PR